$OpenBSD: patch-hcxpcaptool_c,v 1.1 2020/04/07 21:15:24 sebastia Exp $

in 5.3.0 there was a PATH_MAX defined to 255 which went
away, now define it properly only for OpenBSD to what
matches sys/syslimits.h

Index: hcxpcaptool.c
--- hcxpcaptool.c.orig
+++ hcxpcaptool.c
@@ -18,6 +18,9 @@
 #include <openssl/hmac.h>
 #include <openssl/cmac.h>
 #if defined (__APPLE__) || defined(__OpenBSD__)
+#if defined(__OpenBSD__)
+#define PATH_MAX 1024	/* as defined in sys/syslimits.h */
+#endif
 #include <libgen.h>
 #include <sys/socket.h>
 #else
